lib/grandpa/message_handler.go

```diff
@@ -78,8 +78,9 @@ func (h *MessageHandler) handleMessage(from peer.ID, m GrandpaMessage) (network.
 	}
 }
 
-func (h *MessageHandler) handleNeighbourMessage(packet *NeighbourPacketV1, from peer.ID) error {
-	logger.Debugf("handling neighbor message from peer %v with set id %v and round %v", from.ShortString(), packet.SetID, packet.Round)
+func (h *MessageHandler) handleNeighbourMessage(packet *NeighbourPacketV1, from peer.ID) error { //nolint
```

Comment:
Is the `//nolint` for something other than the "neighbour" spelling?

Reply:
Yeah it is because error will always be nill. I didn't want to change this to avoid `return nil, nil` in the caller. But now i see this already happens, so i updated to remove returning an error here